<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDock</class>
 <widget class="QWidget" name="ImageDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>874</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment">
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="lHorizontalAlignment">
         <property name="text">
          <string>Hor. align.:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="chbEmbedded">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="lRotation">
         <property name="text">
          <string>Rotation:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="2">
        <widget class="QComboBox" name="cbVerticalAlignment"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lPosition">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="lPositionXLogicalDateTime">
         <property name="text">
          <string>x:</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QCheckBox" name="chbVisible">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="TimedLineEdit" name="leName"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lOpacity">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="lPositionY">
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
         <property name="text">
          <string>Position Y:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lFileName">
         <property name="text">
          <string>File name:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="cbPositionX">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Horizontal position relative to label's parent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="NumberSpinBox" name="sbPositionX">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="suffix">
            <string> cm</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="lPositionYLogical">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="2">
        <widget class="NumberSpinBox" name="sbPositionYLogical">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="lPositionXLogical">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QCheckBox" name="chbBindLogicalPos">
         <property name="text">
          <string>Logical Coordinates</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lPositionX">
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
         <property name="text">
          <string>Position X:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="NumberSpinBox" name="sbHeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="2">
        <widget class="NumberSpinBox" name="sbWidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="cbPositionY">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Vertical position relative to label's parent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="NumberSpinBox" name="sbPositionY">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="suffix">
            <string> cm</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lEmbedded">
         <property name="text">
          <string>Embedded:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="lVerticalAlignment">
         <property name="text">
          <string>Vert. align.:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="18" column="2">
        <widget class="QComboBox" name="cbHorizontalAlignment"/>
       </item>
       <item row="20" column="2">
        <widget class="QSpinBox" name="sbRotation">
         <property name="suffix">
          <string>°</string>
         </property>
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="leFileName">
           <property name="toolTip">
            <string>Specify the name of the image file</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bOpen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string> Select the image file</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lRatio">
         <property name="text">
          <string>Keep Ratio:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="sbOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="UTCDateTimeEdit" name="dtePositionXLogical"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lSize">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="chbKeepRatio">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lHeight">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="NumberSpinBox" name="sbPositionXLogical">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QCheckBox" name="chbLock">
         <property name="text">
          <string>Locked</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBorder">
      <attribute name="title">
       <string>Border</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>695</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>frontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>frontend/widgets/TimedLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>UTCDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>UTCDateTimeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>leFileName</tabstop>
  <tabstop>bOpen</tabstop>
  <tabstop>chbEmbedded</tabstop>
  <tabstop>sbOpacity</tabstop>
  <tabstop>sbWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>chbKeepRatio</tabstop>
  <tabstop>chbBindLogicalPos</tabstop>
  <tabstop>cbPositionX</tabstop>
  <tabstop>sbPositionX</tabstop>
  <tabstop>cbPositionY</tabstop>
  <tabstop>sbPositionY</tabstop>
  <tabstop>sbPositionXLogical</tabstop>
  <tabstop>dtePositionXLogical</tabstop>
  <tabstop>sbPositionYLogical</tabstop>
  <tabstop>cbHorizontalAlignment</tabstop>
  <tabstop>cbVerticalAlignment</tabstop>
  <tabstop>sbRotation</tabstop>
  <tabstop>chbLock</tabstop>
  <tabstop>chbVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
